scout/server/extensions/loqus_extension.py

```diff
@@ -133,37 +147,73 @@ def get_variant(self, variant_info):
                     variant_info["end_chrom"],
                     "--sv-type",
                     variant_info["variant_type"],
+                    ["--case-count"],
                 ]
             )
 
-        if self.version > 2.4:
-            variant_call.extend(["--case-count"])
-
         output = ""
         try:
-            output = execute_command(variant_call)
+            output = execute_command(cmd)
         except CalledProcessError as err:
             LOG.warning("Something went wrong with loqus")
             raise err
 
         res = {}
         if output:
             res = json.loads(output)
-
-        if self.version < 2.5:
-            res["total"] = self.case_count()
-
         return res
 
+    def search_dictlist(self, key):
+        """Search list of dicts"""
+        for i in self.loqusdb_settings:
+            if i.get("id") == key:
+                return i
+        return None
+
+    def default_setting(self):
+        """Get default loqusdb configuration  """
+        return self.search_dictlist("default")
+
+    def get_bin_path(self, loqusdb_id=None):
+        """Return path to `loqusdb` binary as configured per
+        loqusdb_id or default"""
+        if isinstance(self.loqusdb_settings, list) and loqusdb_id is None:
+            return self.default_setting().get(BINARY_PATH)
+        if isinstance(self.loqusdb_settings, list) and loqusdb_id is not None:
+            try:
+                return self.search_dictlist(loqusdb_id).get(BINARY_PATH)
+            except AttributeError:
+                raise ConfigError("LoqusDB id not found")
+
+    def get_config_path(self, loqusdb_id=None):
+        """Return path to `loqusdb` config arguments  as configured per
+        loqusdb_id or default"""
+        if isinstance(self.loqusdb_settings, list) and loqusdb_id is None:
+            return self.default_setting().get(CONFIG_PATH)
+        if isinstance(self.loqusdb_settings, list) and loqusdb_id is not None:
+            try:
+                return self.search_dictlist(loqusdb_id).get(CONFIG_PATH)
+            except AttributeError:
+                raise ConfigError("LoqusDB id not found")
+
+    def get_configured_version(self, loqusdb_id=None):
+        """Return configured version"""
+        if isinstance(self.loqusdb_settings, list) and loqusdb_id is None:
+            return self.default_setting().get(VERSION)
+        if isinstance(self.loqusdb_settings, list) and loqusdb_id is not None:
+            try:
+                return self.search_dictlist(loqusdb_id).get(VERSION)
+            except AttributeError:
+                raise ConfigError("LoqusDB id not found")
+
```

Comment:
Same like above, also please specify return types in docstring

Reply:
Done!